### PR TITLE
Added notebook for tutorial 2 French translation

### PR DESCRIPTION
**Pull request by r0cketr1kky**
Translated the second tutorial from English to the French language.

Issue #2881 #2782

### CODE REVIEW

File: examples/tutorials/translations/french/Part_02_Intro_to_Federated_Learning.ipynb [hunk 1]
```diff
@@ -0,0 +1,360 @@
+{
```
Comment (LaRiffle) [inline examples/tutorials/translations/french/Part_02_Intro_to_Federated_Learning.ipynb]: <p>Hello!</p><p>Je crois qu'en français on dit "Le Federated Learning", "Un exemple de Federated Learning" etc</p><p>Est-ce que tu peux changer les occurrences dans le texte en ce sens?</p><p>Merci!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (r0cketr1kky) [reply examples/tutorials/translations/french/Part_02_Intro_to_Federated_Learning.ipynb]: Done :)

**Comment by r0cketr1kky** [comment]
Sure, will do that by tonight👍

**Comment by r0cketr1kky** [comment]
You can close this PR @srishilesh 
I've opened a new one with the corrections, here #3320

**Comment by srishilesh** [comment]
Closing this PR and shifting it to PR #3320 with corrections made.

**Pull request closed without merging**